android/assets/jsons/Civ V - Gods & Kings/Nations.json

```diff
@@ -566,8 +566,8 @@
 		"startBias": ["Coast"],
 		"preferredVictoryType": "Domination",
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Holy Isabella, envoy of God, Great Queen of Castille and León, your people celebrate you! Spain is a beautiful and ancient country, a link between Europe and Africa, one shore on the Mediterranean and the other on the mighty Atlantic Ocean. The Spanish culture had taken the best of the Muslim and Christian worlds, and was ready to dominate the world for centuries. Spanish powerful military fleets and brave explorers found and conquered much of the New World. Exploiting a whole new continent and its riches, Spain reached a state of wealth and power that only a few can compete during hundreds of years.",
```

Comment:
Powerful Spanish military fleets
reached a state of power few could compete with for hundreds of years